### ISSUE TITLE
License diff URL incorrect in issue created for close license ID

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#138]
When choosing that a license should match, the issue created has an invalid link to the diff:

```'https://github.com/<github-username>/<repository-name>/blob/master/s8zpewkoff.png'```

See https://github.com/spdx/TEST-LicenseList-XML/issues/69 for an example.

**Comment by goneall** [comment]
@Ugtan Can you take a look at this one when you get a chance?

**Comment by Ugtan** [comment]
> When choosing that a license should match, the issue created has an invalid link to the diff:

'https://github.com/<github-username>/<repository-name>/blob/master/s8zpewkoff.png'

This was likely to happen. I have mentioned it a few times in this [PR](https://github.com/spdx/spdx-online-tools/pull/136), I guess you missed it. The working of the tool is as follow: if a close match is encountered then it takes a SC of the diff and uploads it to a github repository where it can be easily accessible for later use. For the purpose of storing screenshots we would have to create a github repo and put the `name with owner` and `github token` in the `postscreenshot` js file to ensure the working in the production environment. The github repository can be either the spdx official repo or you can create a dummy github account for the purpose of storing the images its completely upto you, As you would have to put github token in the js file. Also, I have my exam tomorrow. I will have a look at the other issues tomorrow evening. Thanks

**Comment by goneall** [comment]
@Ugtan Thanks for the quick update.  I do recall the documentation in the PR, now that you mention it.

I'll set that up and test it out in production.

**Comment by goneall** [comment]
I setup a repo for this purpose: https://github.com/spdx/licenseRequestImages

I tested it out with a token I generated and it works great :)

@Ugtan I invited you as an admin for this repo in case you want to try/test anything out.

**Comment by goneall** [comment]
@Ugtan Is there a way we can move the token to the secrets.py or create a secrets.js file for the token?  I'm afraid I will accidentally commit my token to the source repo.

**Comment by Ugtan** [comment]
@goneall Yeah, I'm not sure of a better way to do the same but creating a secret.js file and hard coding all the sensitive data inside it and adding this file to .gitignore is definitely possible and it would be sufficient IMO.

**Comment by jlovejoy** [comment]
is this still an open issue? sounds similar to something more recent??  @goneall

**Comment by goneall** [comment]
I believe this has been working for a while - closing the issue
